### PR TITLE
[bitnami/mariadb-galera] add externalIPs to service

### PR DESCRIPTION
**Pull request by onedr0p**
**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/mariadb-galera/README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ The following table lists the configurable parameters of the MariaDB Galera char
...
 | `service.clusterIP`                        | Specific cluster IP when service type is cluster IP. Use `None` for headless service                                                                                                                                                                                             | `nil`                                                             |
 | `service.nodePort`                         | Kubernetes Service nodePort                                                                                                                                                                                                                                                      | `nil`                                                             |
+| `service.externalIPs`         | External IP list to use with ClusterIP service type                                                        | `[]`                                                             |
```
Comment (marcosbc) [inline bitnami/mariadb-galera/README.md:80]: Could you fix the indentation of this row to be consistent with the rest of the table?

File: bitnami/mariadb-galera/Chart.yaml [hunk 2]
```diff
@@ -28,4 +28,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-mariadb-galera
   - https://github.com/prometheus/mysqld_exporter
   - https://mariadb.org
-version: 5.2.1
+version: 5.2.2
```
Comment (marcosbc) [inline bitnami/mariadb-galera/Chart.yaml:31]: Since this is a feature I would bump to 5.3.0.

**Review by marcosbc — CHANGES_REQUESTED** [review]
Thanks for the contribution! I added a couple of comments, please have a look at them whenever you have a chance.

File: bitnami/mariadb-galera/README.md (hunk 1 above)
Comment (onedr0p) [reply bitnami/mariadb-galera/README.md]: Done :)

File: bitnami/mariadb-galera/Chart.yaml (hunk 2 above)
Comment (onedr0p) [reply bitnami/mariadb-galera/Chart.yaml]: Also done :)

**Review by marcosbc — APPROVED** [review]
LGTM! Thanks for the contribution!

**Pull request merged**